data_steward/resource_files/validation/participants/abbreviation_state.csv

```diff
@@ -0,0 +1,65 @@
+"abbreviated","unabbreviated"
```

Comment:
This is helpful, but would suggest making it complete using [census reference](https://www.census.gov/library/reference/code-lists/ansi/ansi-codes-for-states.html). Also, I don't think you can have commented lines. A comment where you are loading this data would suffice, it could just reference the source.

Reply:
@nishanthpp93 
I agree with you - I think it is better to have it complete.

I found some discrepancies between our CSV file and the census.

[1] The state abbreviations that exist in our CSV but not in the census reference:
fm
mh
pw
aa
ae
ap

[2] The state abbreviation that exists in the census but not in our CSV:
um

I imagine we should keep the ones in [1], though they are not in the census reference.

Reply:
This is a more complete list from [USPS](https://pe.usps.com/text/pub28/28apb.htm)